aplib.net-demo/Assets/Scripts/Player/MouseLook.cs

```diff
@@ -0,0 +1,63 @@
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
+{
+    [Range(0.05f, 0.30f)][SerializeField] float sensitivity = 0.15f;
+	[Tooltip("Limits vertical camera rotation. Prevents the flipping that happens when rotation goes above 90.")]
+	[Range(0f, 90f)][SerializeField] float yRotationLimit = 88f;
+    [SerializeField] Transform playerBody;
+    Vector2 rotation = Vector2.zero;
+    Vector2 mouseInput;
+    bool showMouse = false;
```

Comment:
Add `private` keywords and lower dash prefix to private variables, following C# conventions.